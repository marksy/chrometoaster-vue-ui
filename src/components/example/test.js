import Vue from 'vue';
import Example from './';

Vue.config.productionTip = false;

describe('<example />', () => {
	const Component = Vue.extend(Example)

	it('does not crash', () => {
		const vm = new Component().$mount()
		expect(vm.$el.textContent).toMatch(/Example/)
	});

	it('matches wrapper snapshot', () => {
		expect(Component).toMatchSnapshot('Component mount');
	});
});
