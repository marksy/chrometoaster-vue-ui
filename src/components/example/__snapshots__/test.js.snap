// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<example /> matches wrapper snapshot: Component mount 1`] = `[Function]`;
